api/src/main/java/org/openmrs/module/mksreports/reports/OutpatientConsultationReportManager.java

```diff
@@ -237,6 +239,7 @@ public ReportDefinition constructReportDefinition() {
 		total.setUnknownGenderIncluded(true);
 		OPDConsult.addColumn(col17, createCohortComposition(total, males), null);
 		OPDConsult.addColumn(col18, createCohortComposition(total, females), null);
+		OPDConsult.addColumn(col23, createCohortComposition(total), null);
```

Comment:
We might find a better name later, but for now please refactor `OPDConsult` → `opdConsult`. The current name is just not compliant to Java coding conventions.

Reply:
Ok